src/components/FiltersDrawer.tsx

```diff
@@ -154,20 +149,29 @@ export const FiltersDrawer = ({ open, onClose }: FiltersDrawerProps) => {
             value={filters?.business_market}
             options={transformOptions(businessMarketMap)}
             onSelect={(business_market) => setFilters((prev) => ({ ...prev, business_market }))}
+            clearValue={() => setFilters((prev) => ({ ...prev, business_market: undefined }))}
           />
         </div>
-        <Button
-          type='primary'
-          className='filters-cta'
-          onClick={() => {
-            dispatch({
-              type: 'filters',
-              payload: filters,
-            })
-            handleClose()
-          }}>
-          Apply Filters
-        </Button>
+        <div className='filters-cta-container'>
+          <Button
+            type='text'
+            className='filters-cta clear'
+            onClick={() => {
+              dispatch({ type: 'clearFilters' })
```

Comment:
Do we really need to update the context for this?

I imagine just updating the local state for the `FiltersDrawer` component is more convenient as we're likely going to have a new set of filters anyways as I said in my other comment

so just `setFilters(initialAdditionalFiltersState)` could do what do you think @MrViometal ?

Reply:
I don't disagree but I don't totally agree also, imagine this scenario where you clear the filters local state but the filters context state is still the same and then you see that they are applied in the through the filter tags, it can get confusing visually.

Reply:
@MrViometal You mean despite them being cleared the old filters can still be seen applied on the dashboard?

Reply:
exactly, if we were to clear just the local state, this is exactly what will happen. 
It will look like that:
<img width="961" alt="Screenshot 2024-06-11 at 12 09 54 PM" src="https://github.com/egytech-fyi/egytech-fyi/assets/32945639/e07996b3-27ac-4aee-bd12-0348ce13eabe">

with the current implementation, the user will have 2 choices:
1. to clear each field individually through the clear icon (this will only change the local state)
<img width="483" alt="Screenshot 2024-06-11 at 12 10 49 PM" src="https://github.com/egytech-fyi/egytech-fyi/assets/32945639/82a06605-dce9-4aef-b3df-62b3ac34a828">

2. to clear all fields through the clear fields button which will in turn clear the global context state in order not to confuse the user.

Reply:
One can argue that the individual clear icons and the clear filters button should behave in the same way, and I can see the merit in this argument tbh, but I would rather not rely solely on my judgment on this one, wdyt?

Reply:
convinced